flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/table/MySqlDeserializationConverterFactory.java

```diff
@@ -115,8 +114,7 @@ public Object convert(Object dbzObj, Schema schema) throws Exception {
 
     private static Optional<DeserializationRuntimeConverter> createArrayConverter(
             ArrayType arrayType) {
-        if (LogicalTypeChecks.hasFamily(
-                arrayType.getElementType(), LogicalTypeFamily.CHARACTER_STRING)) {
+        if (hasFamily(arrayType.getElementType(), LogicalTypeFamily.CHARACTER_STRING)) {
```

Comment:
arrayType.getElementType().is(LogicalTypeFamily.CHARACTER_STRING)

Reply:
It was introduced in Flink 1.15 (https://issues.apache.org/jira/browse/FLINK-24599), so it won't compile with enabled Flink-1.14 profile.